### PR TITLE
Adds extra theme

### PR DESCRIPTION
**Pull request by erooijak**
Hi Laura,

I've worked on integrating the [Nucleus HTML5 template](http://www.downloadwebsitetemplates.co.uk/template/nucleus/) in Cryogen. It adds another possible theme to choose from. An example of it can be seen at http://evalapply.io/. 

I updated the documentation to include three themes and added the code for the template. 

The requirement of www.downloadwebsitetemplates.co.uk is that the footer remains, unless a donation is made. Perhaps that requirement makes this extra theme not very attractive. I described it in your README.

Maybe it is helpful. Thanks anyhow for this great tool.

Kind regards,
Erwin Rooijakkers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +187,7 @@ A Clojure library to provide Markdown rendering to the cryogen-core compiler by
 * [Clojurians](http://blog.clojurians.org)
 * [Gundee and Company](http://www.gundee.com/)
 * [Teamcool Rocks](http://www.teamcool.net/index.html)
+* [Teamcool Rocks](http://www.teamcool.net/index.html)
```
Comment (lacarmen) [inline README.md]: Duplicate site here

File: src/leiningen/new/cryogen/themes/nucleus/html/post-content.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <!--   <blockquote> -->
+  <!--   	<p>"Suspendisse finibus justo eu dapibus maximus. Praesent nulla tellus, maximus non elit sed, pretium mattis lorem. Vivamus consectetur lacus vel tortor tincidunt bibendum. Curabitur eu sem at enim luctus laoreet."</p> -->
+  <!--   	<p class="author">Joe Bloggs (London)</p> -->
+  <!--   </blockquote> -->
+  <!--   <p>Nunc ipsum tortor, venenatis et porta eu, venenatis fermentum diam. Etiam porta arcu congue consequat lobortis. Suspendisse finibus justo eu dapibus maximus. Praesent nulla tellus, maximus non elit sed, pretium mattis lorem. Vivamus consectetur lacus vel tortor tincidunt bibendum. Curabitur eu sem at enim luctus laoreet. Morbi felis enim, aliquam vel vestibulum non, condimentum eu dui. Maecenas nec pretium nunc.</p> -->
+	<!-- </div> -->
```
Comment (lacarmen) [inline src/leiningen/new/cryogen/themes/nucleus/html/post-content.html]: Can you remove this extra stuff please?

**Comment by lacarmen** [comment]
It looks like you've checked in some extra files - `.#404.html` and `.#archives.html` as well.

**Comment by lacarmen** [comment]
I will accept the theme once the above comments are addressed :)

**Comment by erooijak** [comment]
Hi *Carmen ;),

Thank you. I addressed your comments in the latest commits. I also updated the footer to contain a name plus date, removed a reference to a non-existent CSS file and changed some indentation here and there.

One annoyance I have with the theme is that the social media icons go over the menu items if you view in landscape mode on mobile. I did my best to fix this with CSS by trying to keep the social media icons always below the lowest menu list item, but I did not succeed.

css_is_awesome

Kind regards,
Erwin

**Comment by lacarmen** [comment]
Thanks!

**Pull request merged**